src/main/java/org/openmrs/addonindex/scheduled/FetchDetailsToIndex.java

```diff
@@ -175,7 +177,8 @@ String fetchConfigXml(AddOnToIndex addOnToIndex, AddOnVersion addOnVersion) thro
 		return null;
 	}
 	
-	void handleConfigXml(String configXml, AddOnVersion addOnVersion) throws Exception {
+	void handleConfigXml(String configXml, AddOnVersion addOnVersion,
+                         AddOnInfoAndVersions addOnInfoAndVersions, boolean setModuleIdAndPackage) throws Exception {
```

Comment:
Sorry, my previous comment was unclear/misleading. I had meant that you should have this new argument represent "should this method call also set the id and package". However since you're using it the other way (i.e. true means it's already done) then you should change this variable name to be "skipModuleIdAndPackage".